client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -105,6 +106,9 @@
     @Option(names = "--external-authentication", paramLabel = "<externalAuthentication>", description = "Enable external authentication")
```

Comment:
Can we make this flag to accept a parameter instead of adding new property?

Reply:
I wanted to avoid that deliberately

Reply:
Why?

Reply:
I think the less config toggles the better.

Reply:
because the default for external authentication is false and default for strategy is desktop. idk how to make it a "default false with desktop strategy" in a single flag